barretenberg/cpp/src/barretenberg/ultra_honk/ultra_verifier.cpp

```diff
@@ -59,7 +59,7 @@ template <typename Flavor> bool UltraVerifier_<Flavor>::verify_proof(const HonkP
     CommitmentLabels commitment_labels;
 
     // TODO(Adrian): Change the initialization of the transcript to take the VK hash?
-    const auto circuit_size = transcript->template receive_from_prover<uint32_t>("circuit_size");
+    const auto circuit_size = transcript->template receive_from_prover<uint32_t>("instance_size");
```

Comment:
there is no concept of instance in the ultra_verifier so let's stick to circuit_size

Reply:
Ok, so you're saying its fine to use circuit_size instead of instance_size for protogalaxy too?

Reply:
i don't have strong opinions, since this will be refactored soon I think it's fine to have `circuit_size` in some places and `instance_size` in the places where we operate on an instance